hypervisor/arch/x86/configs/dnv-cb2/pci_devices.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef PCI_DEVICES_H_
+#define PCI_DEVICES_H_
+
+#define HOST_PCI_DEV0_BDF	.pbdf.bits = {.b = 0x00U, .d = 0x00U, .f = 0x00U}
+#define HOST_PCI_DEV1_BDF	.pbdf.bits = {.b = 0x03U, .d = 0x00U, .f = 0x01U}
+#define HOST_PCI_DEV2_BDF	.pbdf.bits = {.b = 0x00U, .d = 0x15U, .f = 0x00U}
+#define HOST_PCI_DEV3_BDF	.pbdf.bits = {.b = 0x00U, .d = 0x14U, .f = 0x00U}
+#define HOST_PCI_DEV4_BDF	.pbdf.bits = {.b = 0x03U, .d = 0x00U, .f = 0x00U}
```

Comment:
We said each usage has a clear definition of devices to passthru, such as the logical partition mode. Each device may have different BDF (in different SOC platform), and we can define the BDF in per board header files.  We need to use clear name, not DEV0/1/2/3 which is completely blind to every developer.